ja/news/_posts/2015-03-03-ruby-2-2-1-released.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: news_post
+title: "Ruby 2.2.1 Released"
+author: "hsbt"
+translator: "sorah"
+date: 2015-03-03 03:00:00 +0000
+lang: ja
+---
+
+Ruby 2.2.1 がリリースされました。これは 2.2 シリーズにおける最初の TEENY リリースになります。
+
+このリリースは ffi によるビルドの失敗、また Symbol GC のメモリリーク問題 ([Bug #10686](https://bugs.ruby-lang.org/issues/10686)) が修正されています。
+
+その他詳細は [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_2_1/ChangeLog) を参照してください。
+
+## ダウンロード
+
+* [http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.bz2](http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.bz2)
+
+      SIZE:   13326768 bytes
+      MD5:    06973777736d8e6bdad8dcaa469a9da3
+      SHA256: 4e5676073246b7ade207be3e80a930567a88100513591a0f19fc38e247370065
+      SHA512: af6a8e75a66b953ff33ecbca5111bcf1c6560b6b48b370b700820fcbe91363146c5ac8abd670a14e693b44343ae598bab472ed2902834304c03ffcd9550886d1
+
+* [http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.gz](http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.gz)
+
+      SIZE:   16512208 bytes
+      MD5:    b49fc67a834e4f77249eb73eecffb1c9
+      SHA256: 5a4de38068eca8919cb087d338c0c2e3d72c9382c804fb27ab746e6c7819ab28
+      SHA512: 97b92251c612e40ad84c3ce612b0f410010ed72f564a4223af1c8df3f7c9d7b7acea9d75423f033752902e4829272c5c94496eae7f504d2be7442e44519b8e93
+
+* [http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.xz](http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.xz)
+
+      SIZE:   10320352 bytes
+      MD5:    f7570579539faaa734c02e2857370298
+      SHA256: 3b2852c07fdc12d0ba2b5698c7587ae2b46141539a526d143da5860b55a626c0
+      SHA512: 853517426cd4735bab42298a13cef6f4864b6f8787c3985eaa39a4d1efb40e121bdb51e874b567f0ec339fe001920bcd165067633992b2be30e7804191559151
+
+* [http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.zip](http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.zip)
+
+      SIZE:   18457205 bytes
+      MD5:    fc0a654425419e9397a1a6858a76325d
+      SHA256: 73af1b1a9d5ceebfc129f270652fe81d9f760082c9fb3206a81bb7162b2a4087
+      SHA512: 7150f0b6c6d27e6053ac88309903adf42fce537b8228740b97929712689c5623757aeb8a76dab1126cb70970c860fedb55a28c4660d818845921c0c74f8563f8
+
+## リリースコメント
+
+今回の、またこれまでのリリースに協力してくれた皆様に深く感謝申し上げます。
```

Comment:
This line is not different from the original version.
https://github.com/sorah/www.ruby-lang.org/blob/ja-221/en/news/_posts/2015-03-03-ruby-2-2-1-released.md#release-comment